ground/src/test/java/com/google/android/ground/util/recyclerview/RecyclerViewMatcher.kt

```diff
@@ -1,57 +0,0 @@
-/*
- * Copyright 2024 Google LLC
- *
- * Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *
- *     https://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- */
-package com.google.android.ground.util.recyclerview
-
-import android.content.res.Resources
-import android.view.View
-import androidx.recyclerview.widget.RecyclerView
-import org.hamcrest.Description
-import org.hamcrest.Matcher
-import org.hamcrest.TypeSafeMatcher
-
-fun atPositionOnView(recyclerViewId: Int, position: Int, targetViewId: Int): Matcher<View> =
```

Comment:
but we still have recyclerView in codebase? 

Or should we start migrating all to LazyColumn?

Reply:
As per https://github.com/google/ground-android/issues/1795, we should migrate all layout based code to jetpack compose.